client/a8c-for-agencies/sections/referrals/hooks/use-fetch-referrals.ts

```diff
@@ -9,30 +9,31 @@ export const getReferralsQueryKey = ( agencyId?: number ) => {
 };
 
 const getClientReferrals = ( referrals: ReferralAPIResponse[] ) => {
-	const clients = referrals.reduce< { [ key: string ]: Referral } >( ( acc, referral ) => {
+	const sortedReferrals = referrals.slice().reverse();
+	const clientReferrals = sortedReferrals.map( ( referral ) => {
 		const purchases = referral.products.map( ( product ) => ( {
 			...product,
-			referral_id: referral.id, // referral id is needed for the purchase to be unique
```

Comment:
This is not used anymore. But, let's keep it to link the purchase back to the referral